demo/stories/10-announcement.stories.mdx

```diff
@@ -0,0 +1,234 @@
+import { Story, Meta, html, Preview } from '@open-wc/demoing-storybook';
+
+<Meta title="Intro/Announcement" />
+
+# ING Bank open sources Lion
+## A library for building performant, accessible & flexible Web Components
+
+**TL;DR:** Web development is hard, wether you're making your own components, implementing Design Systems, incorporating accessibility, or adding third party dependencies. Lion aims to make your life a little bit easier for you, by taking the ground work of functional, accessible, and performant components out of your hands!
```

Comment:
Great text in general 👍 

Few nitpicks about this paragraph:
- Maybe 'web development *can* be hard'? It's probably a statement not every dev would agree with? I would also explain why we think it's hard by mentioning the challenges we face when building a core component lib: 
 - accessibility (especially combined with shadow dom) 
 - performance
 - cross browser compatibility
 - generecity (a lot of teams/companies build single purpose /'context specific' ui components and not one size fits all (without bloat/while maintaining modularity) libs that cover a lot of edge cases)
 - maturity (a lib with a large amount of consumers will mature/stabilize over time because many edge cases are discovered by all those users).


- `Lion aims to make your life a little bit easier for you` -> feels like saying the same thing twice. `Lion aims to make your life a bit easier` feels better 
- What does *functional* mean in this context? Functional as in 'not broken'? :) As in 'functional programming'?
Maybe 'feature complete' would be a better fit in this context.

Reply:
These are all really good points 👍

Reply:
Thanks. What also might be missing in the article: mention the benefit of using a web component based library compared to a framework bound one.